vcluster/deploy/topologies/air-gapped.mdx

```diff
@@ -0,0 +1,187 @@
+an `air-gapped` Kubernetes cluster without the platform agent running on the host
+cluster.
+
+## Prerequisites
+
+<BasePrerequisites />
+  - `docker` (check with `docker version`)
+  - An offline license key for vCluster Pro (provided by Loft)
+- Optionally: private docker registry that the installer computer and the
+air-gapped Kubernetes cluster can access (e.g. x-private-registry:5000 or
+gcr.io/x-team)
+- vCluster Pro syncer (control plane `StatefulSet`) pod default resource requirements (may need to be increased based on intended vCluster workloads):
+
+```yaml title="vCluster Pro syncer pod resource requirements"
+  - Requests
+    - memory: 256Mi
+    - cpu: 200m
+    - ephemeral-storage: 400Mi
+  - Limits
+    - memory: 2Gi
+    - ephemeral-storage: 8Gi
+```
+
+### Download vCluster Helm chart
+
+Download https://charts.loft.sh/charts/vcluster-0.20.3.tgz:
+
+```bash title="Download vCluster Helm chart"
+curl -O https://charts.loft.sh/charts/vcluster-0.20.3.tgz
+```
+
+### Download and push required container images
+
+- `ghcr.io/loft-sh/vcluster-pro-fips:0.20.3`
+- `ghcr.io/loft-sh/kubernetes-fips:v1.28.14`
+- `library/alpine:3.20`
+- `ghcr.io/loft-sh/vcluster-hpm:0.1.2`
+
+:::info Note on alpine image replacement
+The `alpine` image may be replaced with any similar image with the following `vcluster.yaml` configuration:
+
+```yaml title="vcluster.yaml configuration for custom image"
+sync:
+  toHost:
+    pods:
+      rewriteHosts:
+        initContainer:
+          image: your-registry/your-image:1.0.0
+```
+
+The image that is used to replace the default image must be able to run the following command:
+
+```yaml
+Command: []string{"sh"},
+Args:    []string{"-c", "sed -E -e 's/^(\\d+.\\d+.\\d+.\\d+\\s+)" + fromHost + "$/\\1 " + toHostnameFQDN + " " + toHostname + "/' /etc/hosts > /hosts/hosts"}
+```
+:::
+
+## Configure vCluster for air-gapped install
+
+The `vcluster.yaml` file holds your vCluster configuration and allows overriding the vCluster Helm chart default values.
+
+The following are some default values that may need to be overridden for a vCluster Pro air-gapped install:
+
+### Set default image registry
+
+Use `defaultImageRegistry` to configure a private image registry.
+
+```yaml title="Default Image Registry configuration"
+controlPlane:
+  advanced:
+    defaultImageRegistry: my-private-registry:5000/vcluster/
+```
+
+vCluster prepends the image registry to all images used by vCluster, such as syncer and Kubernetes. For example, `registry.k8s.io/kube-apiserver:v1.30.2` becomes `my-private-registry:5000/vcluster/kube-apiserver:v1.30.2`.
+
+### Create image pull secret
+
+`imagePullSecrets` defines extra image pull secrets for the vCluster control plane `ServiceAccount`.
+
+```yaml title="Image Pull Secrets configuration"
+controlPlane:
+  advanced:
+    serviceAccount:
+      imagePullSecrets:
+        - name-of-your-image-pull-secret
+```
+
+### Configure API key
+
```

Comment:
Is this really the way to deploy vcluster in airgapped without the platform agent? Are we saying that we can only deploy airgapped vcluster WITH a platform?

Reply:
Maybe need to specify that even though the config is the same as if you were using the platform, in this case, the value of the `apiKey` secret is an air-gapped license.